### PR TITLE
docs: aa-core sendTransaction and sendTransactions

### PR DESCRIPTION
**Pull request by denniswon**

### CODE REVIEW

File: site/packages/aa-core/provider/sendTransaction.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: sendTransaction
```
Comment (avasisht23) [inline site/packages/aa-core/provider/sendTransaction.md:6]: we've been doing this:
```suggestion
      content: ISmartAccountProvider • sendTransaction
```

File: site/packages/aa-core/provider/sendTransaction.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - - meta
+    - property: og:description
+      content: Overview of the sendTransaction method on ISmartAccountProvider
+---
+
+# sendTransaction
+
+This takes an ethereum transaction and converts it into a UserOperation, sends the UserOperation, and waits on the receipt of that UserOperation (ie. has it been mined).
+
+If you don't want to wait for the UserOperation to mine, it's recommended to user [sendUserOperation](./sendUserOperation) instead.
```
Comment (avasisht23) [inline site/packages/aa-core/provider/sendTransaction.md:19]: nit: should we use absolute paths? fine with local, but else we use absolute

File: site/packages/aa-core/provider/sendTransactions.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    - name: description
+      content: Overview of the sendTransactions method on ISmartAccountProvider
+  - - meta
+    - property: og:description
+      content: Overview of the sendTransactions method on ISmartAccountProvider
+---
+
+# sendTransactions
+
+This takes a set of ethereum transactions and converts them into one single UserOperation, sends the UserOperation, and waits on the receipt of that UserOperation (ie. has it been mined). If you don't want to wait for the UserOperation to mine, it's recommended to user [sendUserOperation](./sendUserOperation) instead.
```
Comment (avasisht23) [inline site/packages/aa-core/provider/sendTransactions.md:17]:
```suggestion
This takes a set of ethereum transactions and converts them into one single UserOperation, sends the UserOperation, and waits on the receipt of that UserOperation (i.e. has it been mined). If you don't want to wait for the UserOperation to mine, it's recommended to user [sendUserOperation](./sendUserOperation) instead.
```

Comment (avasisht23) [reply site/packages/aa-core/provider/sendTransactions.md:17]: nit: also same comment here on absolute vs local paths

File: site/packages/aa-core/provider/sendTransactions.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+**NOTE**: The account you're sending the transactions _to_ MUST support batch transactions.
+s
```
Comment (avasisht23) [inline site/packages/aa-core/provider/sendTransactions.md:20]: think i saw you cleaned this up in a pr stacked on this, so should be fine

**Review by avasisht23 — APPROVED** [review]
nits, lgtm

File: site/packages/aa-core/provider/sendTransaction.md (hunk 2 above)
Comment (denniswon) [reply site/packages/aa-core/provider/sendTransaction.md:19]: this is pointing to the `sendUserOperation` doc page, not the code.

**Pull request merged**